packages/tools/src/tools/base/ContourSegmentationBaseTool.ts

```diff
@@ -59,7 +59,7 @@ abstract class ContourSegmentationBaseTool extends ContourBaseTool {
       activeSegmentation.getActiveSegmentationRepresentation(toolGroupId);
 
     if (!activeSegmentationRepresentation) {
-      throw new Error(
+      console.warn(
```

Comment:
You need to actively skip the segmentation specific changes here, otherwise the next line will throw.

Reply:
You could create a non-segmentation contour, but I think that is a bad idea.  Could you revert to the throw?